Content.Server/Body/Systems/MetabolizerSystem.cs

```diff
@@ -138,6 +139,9 @@ private void TryMetabolize(Entity<MetabolizerComponent, OrganComponent?, Solutio
             var list = solution.Contents.ToArray();
             _random.Shuffle(list);
 
+            TryComp<BloodstreamComponent>(solutionEntityUid.Value, out var bloodstream);
+            TryComp<MobStateComponent>(solutionEntityUid.Value, out var state);
+
```

Comment:
Can just cache mobstate by doing: 
```suggestion
            var dead = _mobStateSystem.IsDead(solutionEntityUid.Value);
```